ctk/common/src/main/kotlin/org/codice/compliance/verification/binding/PostFormVerifier.kt

```diff
@@ -0,0 +1,312 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.compliance.verification.binding
+
+import com.jayway.restassured.path.xml.element.Node
+import com.jayway.restassured.response.Response
+import de.jupf.staticlog.Log
+import org.apache.commons.lang3.StringUtils.isNotEmpty
+import org.apache.cxf.rs.security.saml.sso.SSOConstants.RELAY_STATE
+import org.apache.cxf.rs.security.saml.sso.SSOConstants.SAML_RESPONSE
+import org.codice.compliance.SAMLBindings_3_5_3_a
+import org.codice.compliance.SAMLBindings_3_5_3_b
+import org.codice.compliance.SAMLBindings_3_5_4_a
+import org.codice.compliance.SAMLBindings_3_5_4_b
+import org.codice.compliance.SAMLBindings_3_5_4_c
+import org.codice.compliance.SAMLBindings_3_5_4_d
+import org.codice.compliance.SAMLComplianceException
+import org.codice.compliance.debugWithSupplier
+import org.codice.compliance.prettyPrintXml
+import org.codice.compliance.utils.TestCommon
+import org.codice.compliance.utils.TestCommon.Companion.EXAMPLE_RELAY_STATE
+import org.codice.compliance.utils.TestCommon.Companion.IDP_ERROR_RESPONSE_REMINDER_MESSAGE
+import org.codice.compliance.utils.TestCommon.Companion.MAX_RELAY_STATE_LEN
+import org.codice.compliance.utils.TestCommon.Companion.NAME
+import org.codice.compliance.utils.TestCommon.Companion.VALUE
+import org.codice.compliance.utils.extractSamlResponseForm
+import org.codice.security.saml.SamlProtocol.Binding.HTTP_POST
+
+/* Small pieces of exception messages are being picked up on. */
```

Comment:
✏️ I would add that inside the suppression parenthesis. I was confused cause I thought it was a comment on the class and it didn't make sense. 😄